Mu2eG4/src/constructHall.cc

```diff
@@ -122,25 +125,27 @@ namespace mu2e {
 
       geoOptions->loadEntry( config, volume.getName(), volume.getName() );
 
-      if ( notchMgr.hasNotches( volName ) ) {
-        // First do volumes with notches
+      if ( notchMgr.hasNotches( volName )||notchMgr.hasHoles( volName ) ) {
+       vector<Notch> volNotches = notchMgr.getNotchVector(volName);
+       vector<Hole> volHoles = notchMgr.getHoleVector(volName);
 
-        // Make the VolumeInfo, without solid info
+       // First do volumes with notches
+       // Make the VolumeInfo, without solid info
         VolumeInfo tmpVol(volume.getName(),
                           volume.getOffsetFromMu2eOrigin() - hallInfo.centerInMu2e(),
                           hallInfo.centerInWorld);
-
-        // Make the main extruded solid from which notches will be subtracted
+       // Make the main extruded solid from which notches will be subtracted
         G4ExtrudedSolid* aVol = new G4ExtrudedSolid(tmpVol.name,
                                                     volume.getVertices(),
                                                     volume.getYhalfThickness(),
                                                     G4TwoVector(0,0), 1., G4TwoVector(0,0), 1.);
-
-        // Now loop over the notches and subtract them from above
-        // First, create the eventual solid
-        G4SubtractionSolid* aSolid = 0;
-        // Get the vector of notches
-        vector<Notch> volNotches = notchMgr.getNotchVector(volName);
+       //-------------------------------------------------------------------------------------
+       // Now loop over the notches and subtract them from above
+       // First, create the eventual solid
+       G4SubtractionSolid* aSolid = 0;
+       //-------------------------------------------------------------------------------------
+       // Get the vector of notches
+       //vector<Notch> volNotches = notchMgr.getNotchVector(volName);
```

Comment:
Please remove the commented out line.  ( it's Ok for work in progress to be commented out I don't think that's the case here.)  Same for the vector<Hole>, below.

Reply:
removed lines 148 and 179.

Reply:
Thanks